azerothcore/azerothcore-wotlk
# feat(Scripts/Queue): implement .queue command

## Changes Proposed:
- Added new command `.queue`

## Issues Addressed:
- Closes https://github.com/azerothcore/mod-cfbg/issues/65

## Tests Performed:
- 

## How to Test the Changes:
1. Enter queue (bg/random bg/arena skirmish/arena rated)
2. Check command `.queue show` and add the type (bg/arena)

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

## Media
`.queue s b`
![image](https://user-images.githubusercontent.com/18329778/121098160-7953ab80-c81f-11eb-94cb-379baeb19618.png)
`.queue s b`
![image](https://user-images.githubusercontent.com/18329778/121098178-86709a80-c81f-11eb-9b38-8814ff2ca370.png)
`.queue s a n`
![image](https://user-images.githubusercontent.com/18329778/121098188-8a042180-c81f-11eb-9a42-68981ae6193c.png)
`.queue s a r`
![image](https://user-images.githubusercontent.com/18329778/121098200-8d97a880-c81f-11eb-9d66-4b139105ddeb.png)
